packages/orchestrator-ui-components/src/components/WFOForms/CreateForm.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2019-2023 SURF.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import React, { useCallback, useEffect, useState } from 'react';
+import { Form, FormNotCompleteResponse } from '../../types/forms';
+import UserInputFormWizard from './UserInputFormWizard';
+import { apiClient } from '../../api';
+
+interface IProps {
+    /* eslint-disable  @typescript-eslint/no-explicit-any */
+    preselectedInput?: any;
```

Comment:
preselectedInput can be anything, but how about unknown or a key-value kind of type?